code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+
+const ThoughtForm = ({ newThought, onNewThoughtChange, onFormSubmit }) => {
+    return (
+        <div className='container'>
+            <form className='form-container' onSubmit={onFormSubmit}>
+                <div className='new-thoughts-wrapper'>
+                    <label htmlFor='newThought'>What is making you happy right now?</label>
+                    <div>
+                        <textarea
+                            className='textarea'
+                            id='newThought'
+                            type='text'
+                            value={newThought}
+                            onChange={onNewThoughtChange}
+                            placeholder="Type here..."
+                        />
+                        <p className='letter-counter'>
```

Comment:
Very good that you used a function that will tell the user how many characters is left to type. Interesting to see how javascript can be used in React.